spec/tbd_tests_spec.rb

```diff
@@ -990,53 +990,6 @@
     model.save(file, true)
   end
 
-  it "can invalidate KIVA inputs (smalloffice)" do
```

Comment:
Moved the test to the TBD Tests [repo](https://github.com/rd2/tbd_tests/blob/b7b76534538da6241f752b26fcf06614c52cdafb/spec/tbd_tests_spec.rb#L9809).